src/sentry/integrations/jira_server/integration.py

```diff
@@ -78,6 +78,12 @@
         """,
         IntegrationFeatures.ISSUE_SYNC,
     ),
+    FeatureDescription(
+        """
+        Automatically create Jira tickets based on Issue Alert conditions.
```

Comment:
We don't say Jira Server for the other features, so I'm keeping parity by just calling this Jira